internal/watcher/watcher_test.go

```diff
@@ -149,7 +149,7 @@ func TestNewDifferential(t *testing.T) {
 			token:             "",
 		}
 
-		actual, err := NewDifferential(nil, nil, nil, "", "")
+		actual, err := NewDifferential(nil, nil, nil, "", "", "")
```

Comment:
I would love to see a test for multiple big files.

Reply:
@hanzei this changes has been reverted by https://github.com/mattermost/mattermost-plugin-nagios/pull/31/commits/9a459726ef24c32fec41c57f995b74f477cf03e7